backend/src/test/java/codezap/template/service/TemplateServiceTest.java

```diff
@@ -7,272 +7,566 @@
-        CreateTemplateRequest createdTemplate = makeTemplateRequest("title");
-        Template template = saveTemplate(createdTemplate, new Category("category1", member), member);
-        categoryRepository.save(new Category("category2", member));
-
-        // when
-        Member otherMember = MemberFixture.getSecondMember();
-        UpdateTemplateRequest updateTemplateRequest = makeUpdateTemplateRequest(2L);
-
-        // then
-        Long templateId = template.getId();
-        assertThatThrownBy(
-                () -> templateService.updateTemplate(otherMember, templateId, updateTemplateRequest, category))
-                .isInstanceOf(CodeZapException.class)
-                .hasMessage("해당 템플릿에 대한 권한이 없습니다.");
-    }
+            assertThat(actual).hasSize(2)
+                    .containsExactly(template1, template2);
+        }
 
-    @Test
-    @DisplayName("템플릿 삭제 성공: 1개")
-    void deleteTemplateSuccess() {
-        // given
-        Member member = MemberFixture.getFirstMember();
-        CreateTemplateRequest createdTemplate = makeTemplateRequest("title");
-        saveTemplate(createdTemplate, new Category("category1", member), member);
+        @Test
+        @DisplayName("멤버 ID로 템플릿 조회 성공: 해당하는 템플릿이 없는 경우")
+        void getByMemberIdSuccessWithEmptyList() {
+            var memberId = 100L;
 
-        // when
-        templateService.deleteByMemberAndIds(member, List.of(1L));
+            var actual = sut.getByMemberId(memberId);
 
-        // then
-        assertThat(templateRepository.findAll()).isEmpty();
+            assertThat(actual).isEmpty();
+        }
     }
 
-    @Test
-    @DisplayName("템플릿 삭제 성공: 2개")
-    void deleteTemplatesSuccess() {
-        // given
-        Member member = MemberFixture.getFirstMember();
-        CreateTemplateRequest createdTemplate1 = makeTemplateRequest("title1");
-        CreateTemplateRequest createdTemplate2 = makeTemplateRequest("title2");
-        saveTemplate(createdTemplate1, new Category("category1", member), member);
-        saveTemplate(createdTemplate2, new Category("category1", member), member);
-
-        // when
-        templateService.deleteByMemberAndIds(member, List.of(1L, 2L));
-
-        // then
-        assertThat(templateRepository.findAll()).isEmpty();
+    @Nested
+    @DisplayName("검색 기능")
+    class FindAll {
+
+        private Member member1, member2;
+        private Category category1, category2;
+        private Tag tag1, tag2;
+        private Template template1, template2, template3;
+
+        @Test
+        @DisplayName("검색 기능: 회원 ID로 템플릿 조회 성공")
+        void findAllSuccessByMemberId() {
+            saveInitialData();
+            Long memberId = member1.getId();
+            String keyword = null;
+            Long categoryId = null;
+            List<Long> tagIds = null;
+            Pageable pageable = PageRequest.of(0, 10);
+
+            Page<Template> actual = sut.findAll(memberId, keyword, categoryId, tagIds, pageable);
```

Comment:
```suggestion
            Pageable pageable = PageRequest.of(0, 10);

            Page<Template> actual = sut.findAll(memberId, null, null, null, pageable);
```

(다른 메서드도 포함) 변수 할당을 안해도 의미가 충분히 보일 것 같아요!

Reply:
저는 조금 반대입니다. 다른 테스트들도 같이 본다면 변수 할당을 하지 않아도 어떤 변수들에 null이 들어간건지 바로 알 수 있지만, 해당 테스트 코드만을 봤을 때는 알기 힘들다고 생각해요